### ISSUE TITLE
loss: nan

### ISSUE THREAD

**Issue by xisi789** [ylshaooo/keras-zero-shot-detection#1]
When I trained on VOC2007:
Create YOLOv3 model with 9 anchors and 20 classes.
Load weights /home/gpu/projects/XS/keras-zero-shot-detection-master/data/darknet53_weights.h5.
Freeze the first 249 layers of total 252 layers.
Train on 8967 samples, val on 996 samples, with batch size 8.
Epoch 1/10

   1/1000 [..............................] - ETA: 6:02:42 - loss: nan
   2/1000 [..............................] - ETA: 3:50:33 - loss: nan
   3/1000 [..............................] - ETA: 3:10:26 - loss: nan
   4/1000 [..............................] - ETA: 2:49:20 - loss: nan

python:3.5
keras:2.2
tensorflow:1.10

what's wrong with me?

**Comment by harshu94** [comment]
I am also getting the same.
Where you able to solve it.
I am using on custom dataset

**Comment by ZhangYang-UCAS** [comment]
I am also getting the same.
